docs/index.md

```diff
@@ -6,7 +6,7 @@ Cinder is a clean, responsive theme for static documentation sites that are gene
 
 <small><i class="fas fa-highlighter" style="color:#FA023C"></i> **[highlight.js v9.13.1](https://highlightjs.org/) syntax highlighting with support for [over 25 programming languages](/specimen/#language-support)**</small></br>
 <small><i class="fab fa-font-awesome-alt" style="color:#FA023C"></i> **[FontAwesome v5.5.0](https://fortawesome.github.io/Font-Awesome/) icon support**</small></br>
-<small><i class="fas fa-font" style="color:#FA023C"></i> **[smashingly legible type scheme](/specimen/#typography) to get your message out to your users**</small>
```

Comment:
On the website, this is in a sub-directory.  I think that this will remain broken because the URL points to `https://sourcefoundry.org/specimen#typography` instead of `https://sourcefoundry.org/cinder/specimen/#typography`.  Will play with the relative links to try to fix this.  Maybe `./specimen/#typography` will work?

Reply:
In that case, just removing the front forward slash should work. So it becomes `specimen#typograhy`

Reply:
Ty!  I apparently did it the hard way.  I added the period to the front of the relative URL, pushed it, and it works.  Will leave it be as is and can modify it when I touch the file again.  

Thanks again for all of the recent contributions!  Keep them coming if you have other ideas.